.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  workflow_dispatch:
```

Comment:
Yes - for now. I don't know how expensive this will be and didn't want it to be triggered for every push to every PR... Well, I don't even know if this works correctly. Thus, I would like to test it manually before considering to run it automatically.